resources/views/modals/edit_profile_modal.blade.php

```diff
@@ -219,8 +231,10 @@
 @if(Auth::user()->level() >= $constants['access_level']['seniorstaff'])
 <script type="text/javascript">
   var $url_edit_profile_modal_POST = '{{ url('member/edit/edit_user/') }}/';
+  var $url_edit_profile_password_modal_POST = '{{ url('member/edit/edit_user_password/') }}/';
 </script>
+<script src="/assets/vendors/sweetalert/sweetalert.min.js"></script>
```

Comment:
I'd move this over to master/app.blade.php

I see it being used in the future for other stuff as well.